server.js

```
@@ -438,6 +476,128 @@ app.get('/api/admin/produits', checkAuth, checkAdmin, (req, res) => {
+        const produitsContent = fs.readFileSync(produitsPath, 'utf8');
+        
+        // Extraire l'objet produits du fichier JavaScript
+        const produitsMatch = produitsContent.match(/const produits = ({[\s\S]*?});/);
+        if (!produitsMatch) {
+            return res.status(500).json({ success: false, message: 'Format de fichier produits invalide' });
+        }
+        
+        // Évaluer l'objet JavaScript de manière sécurisée
+        const produitsObj = eval('(' + produitsMatch[1] + ')');
+        
+        res.json({ success: true, produits: produitsObj });
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)